### DISCUSSION TITLE
[QUESTION] How to force colorize output if using pager such as less

### DISCUSSION THREAD

**Discussion by conao3** [fastapi/typer#1316] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import typer

def main(name: str):
    typer.echo(typer.style(f"Hello {name}", fg=typer.colors.GREEN))

if __name__ == "__main__":
    typer.run(main)
```

### Description

How to force colorize output if using pager such as `less`?

### command
```sh
$ python main.py typer
Hello typer
```
with green color.
```sh
$ poetry run python main.py typer | cat
Hello typer
```
with plain color.

Can we tell typer force colorize output such as `--color` or `--color=always` option?

### Operating System

Linux

### Typer Version

0.4.0

### Python Version

Python 3.10.1

**Comment by robinbowes** [comment, marked as answer]
typer.echo() is a wrapper around click.echo(), so you can use the same options.

I think the `color` parameter might be helpful. Default is `color=None` ie. auto-detect. You should be able to pass `color=True` I think.
